tests/MassTransit.ActiveMqTransport.Tests/Configure_Specs.cs

```diff
@@ -225,11 +227,27 @@ public async Task Should_connect_locally_with_test_harness_and_publish_without_c
 
         [Test]
         [Category("Flaky")]
-        public async Task Should_do_a_bunch_of_requests_and_responses()
+        [TestCase("activemq")]
+        [TestCase("artemis")]
+        public async Task Should_do_a_bunch_of_requests_and_responses(string flavor)
         {
-            var bus = Bus.Factory.CreateUsingActiveMq(sbc =>
+            var bus = Bus.Factory.CreateUsingActiveMq(cfg =>
             {
-                sbc.ReceiveEndpoint("test", e =>
+                if (flavor == "artemis")
+                {
+                    cfg.Host("localhost", 61618, cfgHost =>
+                    {
+                        cfgHost.Username("admin");
+                        cfgHost.Password("admin");
+                    });
+                    cfg.EnableArtemisCompatibility();
+                    cfg.UpdateReceiveQueueName((generatedName) =>

```

Comment:
yeah, this has to go.

Reply:
new commits have different implementation